src/status_im/contexts/chat/events.cljs

```diff
@@ -47,7 +47,8 @@
 (defn group-chat?
   ([chat]
    (and (multi-user-chat? chat)
-        (not (:public? chat))))
+        (not (:public? chat))
+        (not= (:chat-type chat) constants/community-chat-type)))
   ([cofx chat-id]
```

Comment:
This predicate returned a false positive, since a community-chat is a multi-user-chat and is not public

Reply:
Good catch!

Reply:
This logic of detecting if chat is group looks suspiciously complicated. There is a `constants/private-group-chat-type`. Can't we compare `(:chat-type chat)` to it like we do in `(public-chat?)`

Reply:
Hey @vkjr 

Thanks for the suggestion. I'm aware of that constant, but we have 6 different types of chats and I don't know them all. also I'm not willing to change the condition here since more bugs could arise.

Instead I preferred to just patch to avoid the existing false positive

Reply:
@ulisesmac, bur from you runtime inspection, does the value of `(:chat-type chat)` is equal to `constants/private-group-chat-type` in a case that you fixed?
Also maybe it worth adding some testcases, at least for already made fix if we know that this part is fragile, wdyt?

Reply:
> bur from you runtime inspection, does the value of (:chat-type chat) is equal to constants/private-group-chat-type in a case that you fixed?

I didn't checked for those values.

> Also maybe it worth adding some testcases, at least for already made fix if we know that this part is fragile, wdyt?

Sure, I can add the test case for `group-chat?` fn